src/services/user.service.js

```diff
@@ -6,6 +6,7 @@ export class UserService {
   }
 
   static async updateUser(fields, id) {
-    return await User.update({ ...fields }, { where: { id: id } });
+    await User.update({ ...fields }, { where: { id: id } });
+    return User.findByPk(id);
   }
 }
```

Comment:
This service is specifically to update user info, not find user info

Reply:
done